doc/whats_new/v0.21.rst

```diff
@@ -43,231 +37,21 @@ Support for Python 3.4 and below has been officially dropped.
     section should be ordered according to the label ordering above. Entries
     should end with: :issue:`123456` by :user:`Joe Bloggs <joeongithub>`.
 
-:mod:`sklearn.cluster`
```

Comment:
This still isn't right.

Reply:
Hmm. Can you link me to the version of `v0.21.rst` you'd like? I can just directly paste that into a new commit.